test/vanilla/package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "@types/mocha": "^2.2.39",
     "@types/should": "^8.1.30",
-    "@types/node": "^6.0.66",
+    "@types/node": "7.0.0",
```

Comment:
@amarzavery FYI, more recent node typings make your code fail to compile (something `headers` were earlier typed as `any`, now as `string[] | string` AFAIK)